content/2fa-services/index.md

```diff
@@ -0,0 +1,132 @@
+
+Still with me? Cool. Let's fix that by having a look at some options out there.
+
+## 1. SuperTokens: Simplified 2FA for Developers
+
+There's a gradient between "build auth yourself" (and potentially introduce various footguns), and "go bankrupt and use BigAuth". SuperTokens lies in the middle - you can self-host it, put it up on a cloud provider of your choice, go managed (without us doing the cost-magically-increases rug pull), mix and match, and/or migrate as the need arises. 
+
+Why it might work for you:
+
+- Pre-built UI that doesn't look like it's from cca 2010. It's also fully customizable.
+- Multi-tenant support that works as advertised.
+- TOTP support, for when you absolutely and positively must use Google Authenticator.
+- No charges for dev licenses - which is an article unto itself considering the offering out there, and best left for another time.
+- Easily integrates with a plethora of backend and frontend frameworks out there.
+
+More importantly to 2FA, SuperTokens comes with OTP via SMS and email. The best part? You won’t need a PhD in cryptography to implement it. Plus, the pricing won’t make your CFO faint - it's $0.01 / MAU  for the managed version (with a minimum charge of $100).
+
+## 2. Duo Security
+
+Suppose enterprise is more your jam, and you’re looking for something that scales well and comes with enterprise features without the enterprise complexity. In that case, Duo Security (now a part of Cisco) might be the answer you're looking for. 
+
+Why it might work for you:
+
+- Clean, modern UI.
+- Strong focus on large organizations with an enteprise-grade 2FA.
+- Push notifications, SMS, and biometric verification
+- Free tier (up to 10 users); Pricing starts at $3 per user per month on the Essentials plan.
+
+While it makes a lot of sense to use Duo for enterprise-grade 2FA, it might be a bit too much for smaller organizations. 
+
+## 3. Okta Adaptive MFA
+
+Continuing with the enterprise theme, Okta Adaptive MFA is a great choice for organizations that need to support a wide range of devices and users. It is a bit on the pricier side (6$ per user per month), but if the feature set it offers is what you need, it's a solid option.
+
+Note: Okta comes with a regular MFA offering, too, priced at $3 per user per month.
+
+Why it might work for you:
+
+- Phishing-resistant by design (WebAuthn, Smart Cards, FastPass)
+- Context-aware authentication that actually works
+- Proactive threat detection via ThreatInsight
+- Device security checks
```

Comment:
Adding links to these resources would help. As someone who is not familiar with Okta, "Context-aware authentication" and "ThreatInsight" don't really mean much to me